docs/demos/UAV_tutorial.py

```diff
@@ -0,0 +1,279 @@
+
+
+# %%
+# Setup Initiator class for the Tracker
+# ---------------------------------------
+# This is just an heuristic initiation:
+# Assume most of the deviation is caused by the Bearing measurement error.
+# This is then converted into x, y components using the target bearing. For z,
+# we simply use range*elev_std_dev (ignore any bearing or range related components).
+# Velocity covariances are just based on expected velocity range of targets.
+#
+# **NOTE** - The Extended Kalman filter can be very sensitive to the state
+# initiation. I tried using the default :class:`~.SimpleMeasurementInitiator` but it tended
+# diverge over the course of the track, especially when large bearing measurement
+# covariances.
+from stonesoup.types.state import GaussianState, State
+from stonesoup.types.update import GaussianStateUpdate
+from stonesoup.initiator.simple import SimpleMeasurementInitiator
+from stonesoup.types.track import Track
+from stonesoup.types.hypothesis import SingleHypothesis
+
+class Initiator(SimpleMeasurementInitiator):
+    def initiate(self, detections, **kwargs):
+        MAX_DEV = 400.
+        tracks = set()
+        measurement_model = self.measurement_model
+        for detection in detections:
+            state_vector = measurement_model.inverse_function(
+                            detection)
+            model_covar = measurement_model.covar()
+            
+            el_az_range = np.sqrt(np.diag(model_covar)) #elev, az, range
+            
+            std_pos = detection.state_vector[2, 0]*el_az_range[1]
+            stdx = np.abs(std_pos*np.sin(el_az_range[1]))
+            stdy = np.abs(std_pos*np.cos(el_az_range[1]))
+            stdz = np.abs(detection.state_vector[2, 0]*el_az_range[0])
+            if stdx > MAX_DEV:
+                print('Warning - X Deviation exceeds limit!!')
+            if stdy > MAX_DEV:
+                print('Warning - Y Deviation exceeds limit!!')  
+            if stdz > MAX_DEV:
+                print('Warning - Z Deviation exceeds limit!!')
+            C0 = np.diag(np.array([stdx, 30.0, stdy, 30.0, stdz, 30.0])**2)
+        
+            tracks.add(Track([GaussianStateUpdate(
+                state_vector,
+                C0,
+                SingleHypothesis(None, detection),
+                timestamp=detection.timestamp)
+            ]))
+        return tracks
+            
+   
+prior_state = GaussianState(
+        np.array([[0], [0], [0], [0], [0], [0]]),
+        np.diag([0, 30.0, 0, 30.0, 0, 30.0])**2)
+initiator = Initiator(prior_state, meas_model)
+#initiator = SimpleMeasurementInitiator(prior_state, meas_model)
+
+# %%
+# Setup Deletor for the Tracker
+# -----------------------------
+# In the simple case of 1 target, we never want to delete the track. Because
+# this Deletor is so simple I haven't bothered using a subtype/inheritance.
+# Instead I make use of Python's duck typing. 
+class MyDeleter:
+    def delete_tracks(self, tracks):
+        return {}
```

Comment:
`{}` is empty dictionary. Should be `set()`. Still works however, as set operation will use dictionary keys (which is an empty set :smile:). But to avoid confusion should change to `return set()`.

Reply:
Done.